test/functional/wallet_encryption.py

```diff
@@ -72,20 +73,25 @@ def run_test(self):
 
         # Test timeout bounds
         assert_raises_rpc_error(-8, "Timeout cannot be negative.", self.nodes[0].walletpassphrase, passphrase2, -10)
-        # Check the timeout
-        # Check a time less than the limit
+
+        self.log.info('Check a timeout less than the limit')
         MAX_VALUE = 100000000
         expected_time = int(time.time()) + MAX_VALUE - 600
         self.nodes[0].walletpassphrase(passphrase2, MAX_VALUE - 600)
+        # give buffer for walletpassphrase, since it iterates over all crypted keys
+        expected_time_with_buffer = time.time() + MAX_VALUE - 600
```

Comment:
While touching this test, mind renaming `MAX_VALUE` to `MAX_SLEEP_TIME` (per `src/wallet/rpcwallet.cpp:1884`) and moving the definition up to the top of the file or using local variable naming, e.g. `max_sleep_time`.

Reply:
There are several of both `int(time.time())` and `time.time()` in the test. Standardize on one of the two?

Reply:
They rounding needs to be exactly the way it is, otherwise the test wouldn't pass